chat.go

```diff
@@ -110,72 +156,75 @@ type ChatCompletionMessage struct {
 
 	// For Role=tool prompts this should be set to the ID given in the assistant's prior request to call a tool.
 	ToolCallID string `json:"tool_call_id,omitempty"`
+
+	// If the audio output modality is requested, this object contains data about the audio response from the model.
+	Audio *ChatCompletionAudio `json:"audio,omitempty"`
+}
+
+type chatCompletionMessageMultiContent struct {
+	Role         string               `json:"role"`
+	Content      string               `json:"-"`
+	Refusal      string               `json:"refusal,omitempty"`
+	MultiContent []ChatMessagePart    `json:"content,omitempty"`
+	Name         string               `json:"name,omitempty"`
+	FunctionCall *FunctionCall        `json:"function_call,omitempty"`
+	ToolCalls    []ToolCall           `json:"tool_calls,omitempty"`
+	ToolCallID   string               `json:"tool_call_id,omitempty"`
+	Audio        *ChatCompletionAudio `json:"audio,omitempty"`
+}
+
+type chatCompletionMessageSingleContent struct {
+	Role         string               `json:"role"`
+	Content      string               `json:"content"`
+	Refusal      string               `json:"refusal,omitempty"`
+	MultiContent []ChatMessagePart    `json:"-"`
+	Name         string               `json:"name,omitempty"`
+	FunctionCall *FunctionCall        `json:"function_call,omitempty"`
+	ToolCalls    []ToolCall           `json:"tool_calls,omitempty"`
+	ToolCallID   string               `json:"tool_call_id,omitempty"`
+	Audio        *ChatCompletionAudio `json:"audio,omitempty"`
```

Comment:
Curious if `omitempty` could've worked with Content/MultiContent?

Reply:
It shouldn't cause any problems since unittests passed.